src/routes/HomePage/index.tsx

```diff
@@ -20,17 +20,23 @@ export default function HomePage() {
   }
 
   function guestAccountLogin() {
-    const userName = 'guest';
+    const guestName = Cookies.get('guest_user_name');
 
-    cookie.set('sessionId', nanoid(16), {
-      domain: window.location.hostname,
-      sameSite: 'Strict',
-    });
-    cookie.set('userName', btoa(userName), {
-      domain: window.location.hostname,
-      sameSite: 'Strict',
-    });
-    navigate(`/quizzes/${userName}`);
+    if (guestName) {
+      logIn({
+        userName: guestName,
+        password: `${guestName}Pa$$word!`,
+      });
+    } else {
+      const guestName = `guest_${nanoid()}`;
+
+      signUp({
```

Comment:
I don't think we need to store guest user's data in our DB. It can always have userName = 'guest' & some default password.

Reply:
fixed